### ISSUE TITLE
Weird CPU and Disk usage 

### ISSUE THREAD

**Issue by erinadreno** [microsoft/vscode#147151]
Issue Type: <b>Bug</b>

Vscode seems to be consuming a lot of CPU and disk resources while doing nothing. This only happens after the second time it has been invoked. Whether from the IDE or directly launching the vscode itself. Sometime it does not close or close very slowly. 

VS Code version: Code 1.66.1 (8dfae7a5cd50421d10cd99cb873990460525a898, 2022-04-06T14:50:12.141Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-12700 (16 x 2112)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.85GB (56.19GB free)|
|Process Argv|--crash-reporter-id d2260390-d5d2-4374-ac92-97ff8d43b548|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
latex-workshop|Jam|8.24.1
python|ms-|2022.4.1
vscode-pylance|ms-|2022.4.0
jupyter|ms-|2022.3.1000901801
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6
veriloghdl|msh|1.5.3
code-spell-checker|str|2.1.11
material-theme|zhu|3.13.20

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
pythonvsnew555:30457759
vscscmwlcmt:30465135
cppdebug:30466689
vscaac:30438847
pynewfile477:30463512

```

</details>

**Comment by erinadreno** [comment]
First launch CPU & disk usage
![FirstLaunch](https://user-images.githubusercontent.com/68513313/162582372-241019f7-617f-4290-81cc-f46076ff780f.png)

Second launch
![vscodeSecondLaunchClosing](https://user-images.githubusercontent.com/68513313/162582410-8f47abd8-58b7-42b2-9a71-1905bfe7eda5.png)
![vscodeSecondLaunch3](https://user-images.githubusercontent.com/68513313/162582411-2f08ca07-e1c3-486d-8f51-7d68a20d1126.png)

**Comment by erinadreno** [comment]
The UI is very laggy as well. Almost the same as pure software rendering in Linux

https://user-images.githubusercontent.com/68513313/162582810-5adfef21-949d-462c-b8a0-dd2a22ab2617.mp4

**Comment by erinadreno** [comment]
The UI is very laggy as well. Almost the same as pure software rendering in Linux

https://user-images.githubusercontent.com/68513313/162582858-3e93bff9-93fd-4334-9188-6bd93fa0a847.mp4

**Comment by erinadreno** [comment]
Here's the log file from the problematic launches. Just in case it may help
[main.log](https://github.com/microsoft/vscode/files/8457360/main.log)

**Comment by lorand-horvath** [comment]
I confirm these issues, most probably duplicate of https://github.com/microsoft/vscode/issues/146737

**Comment by deepak1556** [comment]
@erinadreno can you provide the output of `code --status` when the high cpu usage happens.

**Comment by erinadreno** [comment]
@deepak1556 

`code --status ` doesn't seem to provide anything. After a while vscode just crashed and this command still hangs in the terminal. The log still contains the same error message as I mentioned earlier.

![image](https://user-images.githubusercontent.com/68513313/163219622-480505ba-b28f-4d53-b67e-689582c45cb7.png)

**Comment by erinadreno** [comment]
@deepak1556 

The crash is not a given, and possibly not related to the `code --status` command. It just crashed on its own. Nevertheless, here are the logs.

The `code --status --no-sandbox` returns
[codeStatusOutput.txt](https://github.com/microsoft/vscode/files/8483155/codeStatusOutput.txt)

As for the `Get-WmiObject`, it gave me
[getWmiOutput.txt](https://github.com/microsoft/vscode/files/8483158/getWmiOutput.txt)

It seems that somehow this issue only occurs when I invoke vscode from the IDE via
`"C:/Users/shinatose/AppData/Local/Programs/Microsoft VS Code/Code.exe" -g [file name]:[line number]`
And I can spawn multiple such process by closing the GUI and relaunch it
![image](https://user-images.githubusercontent.com/68513313/163229099-0fc495d9-da2a-4b8f-b611-b070333218eb.png)

**Comment by erinadreno** [comment]
This issue has been fixed in 1.67 release.
